main.nf

```diff
@@ -702,7 +703,7 @@ process fastqc {
     """
     } else {
     """
-    fastqc -q $r1
+    fastqc -t ${task.cpus} -q $r1
```

Comment:
Pretty sure I saw this bit in another PR already?

Reply:
Yes, branched off that PR acceidnetly. Will disappear once that PR is merged in :)